src/fluidplayer.js

```diff
@@ -467,7 +467,9 @@ const fluidPlayerClass = function () {
         const videoWrapper = document.getElementById('fluid_video_wrapper_' + playerNode.id);
 
         if (!self.mobileInfo.userOs) {
-            videoWrapper.addEventListener('mouseleave', self.handleMouseleave, false);
+            if (!self.displayOptions.layoutControls.controlBar.autoHide) {
+                videoWrapper.addEventListener('mouseleave', self.handleMouseleave, false);
```

Comment:
Removing this event makes the video title to hide as well, as the setting relates to `controlBar`, this does not seem like the intended behaviour. Settings to reproduce:
```
layoutControls: {
    title: 'My video title', // Default false
    controlBar: {
        autoHide: true, // Default false
        autoHideTimeout: 5, // Default 3
        animated: true // Default true
    }
},
```
Otherwise LGTM.

Reply:
Approving as it's supposed to remove the title also, thanks @MariaAlmeidaMJ